### ISSUE TITLE
"New button" on secrets page does not work on small screens

### ISSUE THREAD

**Issue by lukestephenson-zendesk** [zendesk/samson#3402]
If I split my screen 50/50, the `New` button on https://samson.zende.sk/secrets no longer responds to clicks.

![image](https://user-images.githubusercontent.com/48660846/59572764-a2d71200-90f2-11e9-871d-82e7bc80259a.png)

At full screen it works, but I wanted side by side with some instructions from confluence that I was trying to follow.

**Comment by lukestephenson-zendesk** [comment]
Same on the secret sharing grants screen https://samson.zende.sk/secret_sharing_grants

**Comment by grosser** [comment]
yeah this is all kinda messy, the whole search UI needs some more consistency :(

also the search_form `capture(&block) << content_tag(:div, button, class: "col-md-1 clearfix")` needs to lose the clearfix I think ...

I poked it a bit, but seems not to be easy ... adding z-index did not work either, I think first the search UIs need some more consistency before we go into more details
